### DISCUSSION TITLE
Dark Theme

### DISCUSSION THREAD

**Discussion by parasiteoflife** [jhuckaby/Cronicle#798] [Ideas]
Please add a dark theme, my eyes hurt

**Comment by jhuckaby** [comment]
So sorry -- it's coming in v2!

**Comment by mikeTWC1984** [comment]
@jhuckaby  isn't it a part of your app framework though? 
Like if you do ```app.toggleTheme()```  in console it will switch the theme.

**Comment by jhuckaby** [comment]
@mikeTWC1984 Well, it kinda does.  The problem is I added dark mode to my app framework years after Cronicle was released, and as a result it isn't really compatible.  Meaning, there are a bunch of little things all over the CSS that have "hard-coded" light colors for various things.  Stuff like category colors, all the graphs and charts (which are a separate library), the hovering clock widget, the live log view, the completed log view, user profile pictures, the password strength indicator, and a bunch of other crap all over the place.

![Screenshot](https://pixlcore.com/screenshots/Screenshot_2024-08-26_at_9.25.11_AM.png)

![Screenshot](https://pixlcore.com/screenshots/Screenshot_2024-08-26_at_9.27.02_AM.png)

It is doable, it would just take a lot of time to clean everything up, and pick nice dark colors for things like categories, and I just really wanted to focus my time on Cronicle v2.

**Reply by mikeTWC1984** [reply]
OK, make sense. Although I'd say if you make that job log font color gray-ish, it will work OK on both themes. Other items might look weird on dark theme, but kinda usable. So if user wants they can do a hack with app.toggleTheme().  On some monitors classic theme burns eyes indeed :)

**Comment by iamumairayub** [comment]
@parasiteoflife 

https://github.com/iamumairayub/cronicle_dark_theme
